repository.json

```diff
@@ -678,6 +678,34 @@
             "version": "1.9.7",
             "url": "https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-1.9.7-windows.zip",
             "fileMatchInside": ".*/phantomjs.exe$"
+        },
+        {
+            "name": "marionette",
+            "platform": "windows",
+            "bit": "32",
+            "version": "0.8.0",
+            "url": "https://github.com/mozilla/geckodriver/releases/download/v0.8.0/geckodriver-v0.8.0-win32.zip"
+        },
+        {
+            "name": "marionette",
+            "platform": "linux",
+            "bit": "64",
+            "version": "0.8.0",
+            "url": "https://github.com/mozilla/geckodriver/releases/download/v0.8.0/geckodriver-0.8.0-linux64.gz"
+        },
+        {
+            "name": "marionette",
+            "platform": "mac",
+            "bit": "32",
+            "version": "0.8.0",
+            "url": "https://github.com/mozilla/geckodriver/releases/download/v0.8.0/geckodriver-0.8.0-OSX.gz"
+        },
+        {
+            "name": "marionette",
+            "platform": "mac",
+            "bit": "64",
+            "version": "0.8.0",
+            "url": "https://github.com/mozilla/geckodriver/releases/download/v0.8.0/geckodriver-0.8.0-OSX.gz"
         }
```

Comment:
@andidev 

> Remove the bit detection from the plugin and enforce the user to set it.

This does not work. I observe many dev workstations to be linux/mac and ci systems like jenkins to be linux. It is not sure to have the same bit combination on both systems.

I'd prefer a special case handling for the special case "iedriver".

What about that?